docs/tutorial/instrument-machine-charms.md

```diff
@@ -1,2 +1,348 @@
 # Instrumenting machine charms
 
+## Prerequisites
+
+- A charmed application that is running in a virtual, or physical, machine.
+- The Canonical Observability Stack, running on Kubernetes.
+
+## Introduction
+
+This tutorial will teach you how to integrate a charm deployed on a machine substrate with the Canonical Observability Stack, running on Kubernetes.
+
+The Grafana Agent machine charm handles installation, configuration, and Day 2 operations specific to [Grafana Agent](https://grafana.com/oss/agent/), using [Juju](https://juju.is). The charm is designed to run in virtual machines as a [subordinate](https://discourse.charmhub.io/t/subordinate-applications/1053). 
+
+```{note}
+Application units are typically run in an isolated container on a machine with no knowledge or access to other applications deployed onto the same machine. With subordinate charms, units of different applications to be deployed into the same container and to have knowledge of each other. Subordinate units scale together with their principal.
+```
+
+## Ensure COS Lite is up and running.
+
+Before we get started, we will make sure that the observability stack is up and running in our `cos` model (follow the tutorial on [getting started with COS Lite](/tutorial/installation/getting-started-with-cos-lite)) in a K8s controller, like this:
+
+```text
+ juju status --relations
+
+Model  Controller  Cloud/Region        Version  SLA          Timestamp
+cos    charm-dev   microk8s/localhost  2.9.42   unsupported  15:43:36-04:00
```

Comment:
This model is not exposing the offers we will later consume from the machine model

https://charmhub.io/topics/canonical-observability-stack/tutorials/instrumenting-machine-charms#step-1-make-sure-cos-lite-is-up-and-running

Reply:
good catch, thanks!